### PR TITLE
Personal slime plush

### PR DESCRIPTION
**Pull request by AjexRose**
## About the PR

added the eris rose slime plush personal item for eris
## Why / Balance

for rp reasons and because there is a criminal lack of slime love
## Technical details

might conflict with other pending personal trinket items
also changed the plush to be semi-similar to lizard plushies (and all that parents it), to be blendable for slime instead of weh juice
## Media

https://github.com/user-attachments/assets/f420d6df-2283-49f9-aa01-91d92e003e8b

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl: AjexRose
- add: Added the eris rose slime plushie as a personal trinket

### CODE REVIEW

File: Resources/Prototypes/_Umbra/Entities/Objects/Fun/toys.yml [hunk 1]
```diff
@@ -73,3 +73,29 @@
...
+  suffix: PersonalItem
+  components:
+  - type: Sprite
+    state: plushie_eris
+    sprite: _Umbra/Objects/Fun/toys.rsi
+  - type: Extractable
+    juiceSolution:
+      reagents:
+      - ReagentId: Slime
+        Quantity: 30
```
Comment (TsjipTsjip) [inline Resources/Prototypes/_Umbra/Entities/Objects/Fun/toys.yml:92]: While this is funny, this adds functionality compared to PlushieSlime and this is not permitted for a personal item.

File: Resources/Prototypes/_Umbra/Entities/Objects/Fun/toys.yml [hunk 2]
```diff
@@ -73,3 +73,29 @@
...
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 20
+        reagents:
+        - ReagentId: Fiber
+          Quantity: 10
+        - ReagentId: Slime
+          Quantity: 10
```
Comment (TsjipTsjip) [inline Resources/Prototypes/_Umbra/Entities/Objects/Fun/toys.yml:101]: Same here.

**Review by TsjipTsjip — CHANGES_REQUESTED** [review]
Two comments, rest is good to merge.

**Comment by TsjipTsjip** [comment]
Merge conflicts resolved.

**Review by TsjipTsjip — APPROVED** [review]
Cool!

Cause non-trivi change, waiting for tests to complete.

**Comment by AjexRose** [comment]
:thumbsup:

**Pull request merged**